src/main/java/de/tum/in/www1/artemis/service/util/VCSSimulationUtils.java

```diff
@@ -0,0 +1,20 @@
+package de.tum.in.www1.artemis.service.util;
+
+import java.util.Random;
+
+public class VCSSimulationUtils {
```

Comment:
this should only be available in DEV mode, please use a profile here

Reply:
I added the dev profile annotation